CelesteNet.Server.ChatModule/CMDs/ParamArgs/ParamPlayerSession.cs

```diff
@@ -36,7 +36,8 @@ public class ParamPlayerSession : Param {
                         // check for partial name elsewhere
                         env.Server.Sessions.FirstOrDefault(session => session.PlayerInfo?.FullName.StartsWith(raw, StringComparison.InvariantCultureIgnoreCase) ?? false);
                 }
-            } else if (uint.TryParse(raw, out playerID)) {
+            }
+            if (session == null && uint.TryParse(raw, out playerID)) {
```

Comment:
This is unrelated to the PR and restores the intended behavior of being able to specify players for commands by their session number. (I guess could be useful when doing a mod action from in-game)